### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by ebenenglish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,65 @@
...
+  - [More Information](#more-information)
+  - [Contact](#contact)
 
-## Documentation
-We are currently working on adding and updating documentation on our [Project Wiki](https://github.com/marriott-library/newspaper_works/wiki)
+<!-- /TOC -->
 
 # Overview
-The Newspaper Works gem provides work type models and administrative functionality for Hyrax-based Samvera applications in the space of scanned newspaper media.  This gem can be included in a Digital Asset Management application based on Hyrax 2.5.1
+NewspaperWorks is a gem (Rails "engine") for [Hyrax](http://hyrax.samvera.org/) -based digital repository applications to support ingest, management, and display of digitzed newspaper content.
```
Comment (brianmcbride) [inline README.md]: Update hyrax link to https

File: README.md [hunk 2]
```diff
@@ -1,64 +1,65 @@
...
 
-## Purpose, Use, and Aims
-This gem, while not a stand-alone application, can be integrated into an application based on Hyrax 2.5 easily to support a variety of cases for management, ingest, and archiving of primarily scanned (historic) newspaper archives.
+NewspaperWorks is not a stand-alone application. It is designed to be integrated into a new or existing Hyrax (2.5.x) application, providing content models, ingest workflows, and feature-rich UX for newspaper repository use-cases.
 
-## Development Status
-This gem is currently under development. The development team is actively working on this project and is updating the codebase nightly. We are targeting an initial 1.0 release for June 2019.
+NewspaperWorks supports:
+* models for Title, Issue, Page, and Article
+* batch ingest via command line
```
Comment (brianmcbride) [inline README.md]: Do we want to specifically mention via rake task?

File: README.md (hunk 1 above)
Comment (ebenenglish) [reply README.md]: Fixed in https://github.com/marriott-library/newspaper_works/pull/184/commits/273880d4612d69bd46c9817b6ca4523fa669826e

File: README.md (hunk 2 above)
Comment (ebenenglish) [reply README.md]: I think this is OK as is, this is the super-condensed feature list, don't think we need a ton of detail here?

**Comment by brianmcbride** [comment]
Looks good to me.

**Pull request merged**